how-to/live-performance.rst

```diff
@@ -3,8 +3,8 @@
 How to fine-tune your server's performance
 ==========================================
 
-In the Divio Cloud Control Panel for your project, you can easily change the resources (instances,
-RAM, storage, transfer) allocated to your project to adjust for its needs.
+In the Divio Cloud Control Panel of your project, you can easily change the resources (instances,
```

Comment:
`In the Divio Dashboard for your project, you can change...`

Reply:
Done